### PR TITLE
completed the ticket number 9

### PR DESCRIPTION
**Pull request by ElraSala14**
completed the ticket number 9

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -218,4 +218,33 @@ describe("GET /api/articles", () => {
...
+      })
+  })
+  test("GET 404: respond with error if the article is not found with the passed article_id", () => {
+    return request(app)
+      .get("/api/articles/11111111/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("The article is not found");
+      });
+  });
```
Comment (Poonam-raj) [inline __tests__/app.test.js:249]: Need a test for a article which is valid but has no comments in the database aka 200 OK with an empty array

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Few more tests needed Salah, keep going you've got this!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -218,4 +218,46 @@ describe("GET /api/articles", () => {
...
+  
+  test("GET 200: resones with an empty array when passed an valid article but has no comments in the database", () => {
+
+    return request(app)
+    .get("/api/articles/4/comments")
+    .expect(200)
+    .then(({body}) => {
+    expect(body.comments).toEqual([])
+
+    })
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js:261]: I would add another 404 test for an article that doesn't exist. For example /api/articles/9999/comments

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
Just add the test I commented on and send another PR :)

File: __tests__/app.test.js (hunk 2 above)
Comment (ElraSala14) [reply __tests__/app.test.js:261]: Hi Rayhaan 
If I added the test shall I wait until my PR is approved?

**Review by karlaferry — APPROVED** [review]
Nice one, Salah! 👏🏼
